src/pages/en/core-concepts/framework-components.md

```diff
@@ -87,7 +87,7 @@ the user scrolls down and the component is visible on the page -->
 
 ### Available Hydration Directives
 
-There are serveral hydration directives available for UI framework components: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` and `client:only=" "`.
+There are serveral hydration directives available for UI framework components: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` and `client:only="FRAMEWORK"`.
```

Comment:
```suggestion
There are serveral hydration directives available for UI framework components: `client:load`, `client:idle`, `client:visible`, `client:media={QUERY}` and `client:only={FRAMEWORK}`.
```
I'm not sure this is best, but I think it's better. Open to suggestions? Our Template Directives page uses `client:only={string}` and `client:media={string}` so it seems like these two directives should at least match?

Reply:
This is tricky because you’d most likely be writing `client:only="react"` in actual code. That applies for both of these but I guess there is a chance you have a file that exports your breakpoints and then pass a variable to `client:media` whereas `client:only` is always just going to be a string. I do agree consistency makes sense here though 👍

Reply:
Yes, and I could even get behind `client:only="framework"` (lowercase) because you won't be typing REACT when you use it.  :)  ... but then maybe that means `client:media="query"` and I don't know how THAT looks...

I don't know how "legal" it would be to do "*framework*"  and "*query*" italicized...? Now I'm just making things up.

Reply:
Haha, don’t think we can italicize inside code? I’d say go with your `{FRAMEWORK}` for now and hope folks understand that’s a placeholder syntax.

Reply:
I thought of "string" but that's not precise enough. Need semantics not syntax. Caps seemed to be how you were representing a placeholder so I followed. Not my favourite or very common :( mabe use <framework> as good old Backus–Naur form but might be confused with HTML markup :(

Reply:
`{FRAMEWORK}` is what should go here